scripts/openpose.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
```

Comment:
Should we launch openpose from a service? Since at the tournament we experienced that the battery drained extremely fast. Would there be a problem with launching it together with the other image recognition software?